bin/utils/subkey/src/main.rs

```diff
@@ -270,7 +270,24 @@ fn get_uri(match_name: &str, matches: &ArgMatches) -> String {
 	}
 }
 
-fn execute<C: Crypto>(matches: ArgMatches)
+#[derive(derive_more::Display, derive_more::From)]
```

Comment:
Could you please change the function above to return an error as well? (get_uri)